components/landing/TheQuickStart/ProvidersList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <section class="providers">
+    <h4>Providers</h4>
+    <ul class="providers__list">
+      <li v-for="item in providersList" :key="item.id">
+        <a class="providers__list__item" :href="item.link" target="_blank">
+          {{ item.title }}
+          <span class="providers__list__cta">
+            <span class="providers__list__cta-label">Learn more</span>
+            <Launch16 class="providers__list__cta-icon" />
+          </span>
+        </a>
+      </li>
+    </ul>
+    <AppCta
+      class="providers__btn"
+      v-bind="providersPage"
+    />
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { GeneralLink } from '~/constants/appLinks'
+
+type ProviderLink = {
+  title: string,
+  id: string,
+  link: string
+}
+
+@Component
+export default class StartLocally extends Vue {
+  @Prop(Array) providersList!: ProviderLink[]
+
+  providersPage: GeneralLink = {
```

Comment:
It would be interesting to have segment here

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3020/commits/dd016e2d83328c7f55e3f7b8f06cd8eb777cdb24